1-source-code/2-writing/2-gofmt-and-friends.md

```diff
@@ -113,31 +113,31 @@ Cool, right?
 
 ## More than formatting
 
-Once gophers were used to having a tool chaing their code on save, many other tools appear
+Once gophers were used to having a tool change their code on save, many other tools appeared
 that made our coding more efficient. Let's see some of them by order of popularity.
 
 These tools differ from `gofmt` in that they modify your code from something that *almost*
 compiles to valid source code.
 
 ### `goimports`: automating import statements
 
-Go is very strict regarding missing or even unused import statements. This cause enough
+Go is very strict regarding missing or even unused import statements. This causes enough
 annoyance to a part of the community that they decided to automate the management of
 import statements.
 
-`goimports` will remove import statements for any packages that are not used in the code
-below. Even cooler, `goimports` will also find references to packages that are not imported
+`goimports` will remove import statements for any packages that are not used in the code.
+Even cooler, `goimports` will also find references to packages that are not imported
 and add them.
 
-How does this magic work, you say? It's quite straight forward, given an expression such as
+You might ask how this magic works. It's quite straightforward. Given an expression such as
```

Comment:
Rather than changing @campoy's voice here I would just change the word "say" to "ask".